app/src/main/java/com/keylesspalace/tusky/components/systemnotifications/NotificationFetcher.kt

```diff
@@ -72,69 +71,41 @@ class NotificationFetcher @Inject constructor(
                     //   (and should therefore adhere to the notification config).
                     eventHub.dispatch(NewNotificationsEvent(account.accountId, notifications))
 
-                    // There's a maximum limit on the number of notifications an Android app
```

Comment:
`notifications` does not need to be mutable any more

(why doesn't GitHub let me comment on unchanged lines 🙄 )

Reply:
ok